src/amr/data/particles/refine/split.h

```diff
@@ -146,7 +234,7 @@ namespace amr
             }
```

Comment:
on line 217 (and above) we should add exception because splitting with 5 babies is only ok for interporder = 3. for smaller interporder, we should never end there.

Reply:
I have left this class entirely in place as is as the reference to base the code in the the specific dimension split files.
just renamed Split to _Split to switch class in use.

Reply:
each implementation needs to manually be exposed via pybind, so the type either exists or it doesn't